lib/components/Widgets/WidgetContainer/index.tsx

```diff
@@ -29,21 +29,25 @@ const Container = forwardRef<
   WidgetContainerProps & { children: ReactNode }
 >(({ header, children }, ref) => (
   <Card ref={ref}>
-    <CardHeader>
-      <div className="flex flex-1 flex-col truncate">
-        <div className="flex flex-row justify-between">
-          <div className="flex min-h-6 grow flex-row items-center gap-1.5 truncate">
-            <CardTitle>{header.title}</CardTitle>
-            {header.subtitle && <CardSubtitle>{header.subtitle}</CardSubtitle>}
-            {header.info && <CardInfo content={header.info} />}
+    {header && (
+      <CardHeader>
+        <div className="flex flex-1 flex-col truncate">
+          <div className="flex flex-row justify-between">
+            <div className="flex min-h-6 grow flex-row items-center gap-1.5 truncate">
+              {header?.title && <CardTitle>{header?.title}</CardTitle>}
+              {header?.subtitle && (
+                <CardSubtitle>{header?.subtitle}</CardSubtitle>
+              )}
+              {header?.info && <CardInfo content={header?.info} />}
+            </div>
+            {header?.link && (
+              <CardLink href={header.link.url} title={header.link.title} />
+            )}
           </div>
-          {header.link && (
-            <CardLink href={header.link.url} title={header.link.title} />
-          )}
+          {header?.comment && <CardComment>{header.comment}</CardComment>}
```

Comment:
The truthy value of header is being checked in 32 already right? Looks like the code might be checking the same value several times.